mlflow/fastai.py

```diff
@@ -135,15 +142,22 @@ def save_model(fastai_learner, path, conda_env=None, mlflow_model=None,
     with open(os.path.join(path, conda_env_subpath), "w") as f:
         yaml.safe_dump(conda_env, stream=f, default_flow_style=False)
 
-    pyfunc.add_to_model(mlflow_model, loader_module="mlflow.fastai",
-                        data=model_data_subpath, env=conda_env_subpath)
+    pyfunc.add_to_model(
+        mlflow_model, loader_module="mlflow.fastai", data=model_data_subpath, env=conda_env_subpath,
```

Comment:
BTW, when running `black --line-length=100 --exclude=mlflow/protos .` with the version in lint-requirements.txt, the result actually differs slightly from the one generated by this PR, e.g. in my local checkout this line does not have the trailing comma:

```
-        mlflow_model, loader_module="mlflow.fastai", data=model_data_subpath, env=conda_env_subpath,
+        mlflow_model, loader_module="mlflow.fastai", data=model_data_subpath, env=conda_env_subpath
```

And I have the following installed:

```
(mlflow-python3) ➜  mlflow git:(master) ✗ which black
/Users/sid.murching/miniconda3/envs/mlflow-python3/bin/black
(mlflow-python3) ➜  mlflow git:(master) ✗ which pip     
/Users/sid.murching/miniconda3/envs/mlflow-python3/bin/pip
(mlflow-python3) ➜  mlflow git:(master) ✗ pip freeze | grep black
black==19.10b0
```

Is the same reproducible for you if you ``pip install -r travis/lint-requirements.txt``?

Reply:
I was able to reproduce the same behavior (no trailing comma) with `black, version 19.10b0`.

Reply:
I checked out all the python files to master and applied black again: https://github.com/mlflow/mlflow/pull/3191/commits/f44b6905554786416b221a55726e601a26a00659

Reply:
I figured out why this is happening. Before running `black --line-length=100 --exclude=mlflow/protos .`, I already ran `black --exclude=mlflow/protos .`

```
# start

pyfunc.add_to_model(mlflow_model, loader_module="mlflow.fastai",
                    data=model_data_subpath, env=conda_env_subpath)

---------------------------------------------------------------------

# after `black --exclude=mlflow/protos .`

pyfunc.add_to_model(
    mlflow_model,
    loader_module="mlflow.fastai",
    data=model_data_subpath,
    env=conda_env_subpath,
)

---------------------------------------------------------------------

# after `black --line-length=100 --exclude=mlflow/protos .`

pyfunc.add_to_model(
    mlflow_model, loader_module="mlflow.fastai", data=model_data_subpath, env=conda_env_subpath,
)
```